cfme/fixtures/service_fixtures.py

```diff
@@ -64,21 +65,28 @@ def create_catalog_item(appliance, provider, provisioning, vm_name, dialog, cata
     return catalog_item
 
 
-@pytest.fixture
+@pytest.yield_fixture
 def order_service(appliance, provider, provisioning, vm_name, dialog, catalog, request):
     """ Orders service once the catalog item is created"""
 
     if hasattr(request, 'param'):
         param = request.param
-        catalog_item = create_catalog_item(provider, provisioning, vm_name, dialog, catalog,
+        catalog_item = create_catalog_item(appliance, provider, provisioning,
+                                           vm_name, dialog, catalog,
                                            console_test=True if 'console_test' in param else None)
     else:
-        catalog_item = create_catalog_item(provider, provisioning, vm_name, dialog, catalog)
+        catalog_item = create_catalog_item(appliance, provider, provisioning,
+                                           vm_name, dialog, catalog)
     service_catalogs = ServiceCatalogs(appliance, catalog_item.catalog, catalog_item.name)
     service_catalogs.order()
     provision_request = appliance.collections.requests.instantiate(catalog_item.name,
-        partial_check=True)
+                                                                   partial_check=True)
```

Comment:
I think this line is over-indented. Please check, is it right?

Reply:
Yes. It's correct

Reply:
Both are correct in [PEP 8's indentation rules|https://www.python.org/dev/peps/pep-0008/#indentation]

Reply:
We do prefer/enforce the hanging indent being lined up in that case though, so this change is correct.